src/languages/fortran.js

```diff
@@ -71,9 +71,9 @@ function(hljs) {
       },
       {
         className: 'number',
-        begin: '-?(\\d+(\\.\\d*)?|\\.\\d+)([DdEe][+-]?\\d+)?',
+        begin: '(?=\\b|\\+|\\-|\\.)(?=\\.\\d|\\d)(?:\\d+)?(?:\\.?\\d*)(?:[de][+-]?\\d+)?\\b\\.?',
```

Comment:
Is it necessary to include points around number into regex? It does not included in tests and it looks like we can skip them.